src/phoenix/server/api/mutations/evaluator_mutations.py

```diff
@@ -204,6 +229,101 @@ async def create_llm_evaluator(
+        user_id: Optional[int] = None
+        assert isinstance(request := info.context.request, Request)
+        if "user" in request.scope:
+            assert isinstance(user := request.user, PhoenixUser)
+            user_id = int(user.identity)
+
+        try:
+            evaluator_name = IdentifierModel.model_validate(input.name)
+        except ValidationError as error:
+            raise BadRequest(f"Invalid evaluator name: {error}")
+
+        output_config = _to_pydantic_categorical_annotation_config(input.output_config)
+
+        try:
+            prompt_version = input.prompt_version.to_orm_prompt_version(user_id)
+        except ValidationError as error:
+            raise BadRequest(str(error))
+
+        try:
+            evaluator_rowid = from_global_id_with_expected_type(
+                global_id=input.evaluator_id,
+                expected_type_name=LLMEvaluator.__name__,
+            )
+        except ValueError:
+            raise BadRequest(f"Invalid LLM evaluator id: {input.evaluator_id}")
+
+        async with info.context.db() as session:
+            llm_evaluator = await session.scalar(
+                select(models.LLMEvaluator)
+                .where(models.LLMEvaluator.id == evaluator_rowid)
+                .options(
+                    joinedload(models.LLMEvaluator.prompt).joinedload(models.Prompt.prompt_versions)
+                )
+            )
+            if llm_evaluator is None:
+                raise NotFound(f"LLM evaluator with id {input.evaluator_id} not found")
+
+            llm_evaluator.name = evaluator_name
+            llm_evaluator.description = (
+                input.description if isinstance(input.description, str) else None
+            )
+            llm_evaluator.output_config = output_config
+
+            # todo: compare against active prompt version as determined by prompt tag or version
+            # https://github.com/Arize-ai/phoenix/issues/10142
+            active_prompt_version = llm_evaluator.prompt.prompt_versions[-1]
+            create_new_prompt_version = not active_prompt_version.has_identical_content(
+                prompt_version
+            )
+            if create_new_prompt_version:
+                llm_evaluator.prompt.prompt_versions.append(prompt_version)
+
+            try:
+                await session.flush()
+            except (PostgreSQLIntegrityError, SQLiteIntegrityError):
+                raise Conflict("An evaluator with this name already exists")
+
+        return LLMEvaluatorMutationPayload(
+            evaluator=LLMEvaluator(id=llm_evaluator.id, db_record=llm_evaluator),
+            query=Query(),
+        )
+
+    @strawberry.mutation(permission_classes=[IsNotReadOnly, IsNotViewer, IsLocked])  # type: ignore
+    async def delete_evaluators(
+        self, info: Info[Context, None], input: DeleteEvaluatorsInput
+    ) -> DeleteEvaluatorsPayload:
+        evaluator_rowids_to_gids: dict[int, GlobalID] = {}
```

Comment:
nit. given the other comments, this can just be a set.
```suggestion
        evaluator_rowids_to_gids: set[int] = set()
```